packages/angular_devkit/build_angular/src/builders/browser-esbuild/schema.json

```
@@ -397,6 +402,10 @@
             }
           },
           "required": ["input"]
+        },
+        {
+          "type": "null",
```

```suggestion
          "const": false,
```
Will allow for `--no-index` on the command line.

---

Done, thanks for reminding me.